pkg/webhook/policy.go

```diff
@@ -161,7 +163,21 @@ func (h *validationHandler) Handle(ctx context.Context, req admission.Request) a
 	if len(res) != 0 {
 		var msgs []string
 		for _, r := range res {
-			if r.EnforcementAction == "deny" {
+			if r.EnforcementAction == "deny" || r.EnforcementAction == "dryrun" {
+				if(*logDenies) {
+					log.WithValues(
+						"process", "admission",
+						"event_type", "violation",
+						"constraint_name", r.Constraint.GetName(),
+						"constraint_kind", r.Constraint.GetKind(),
+						"constraint_action", r.EnforcementAction,
+						"resource_kind", req.AdmissionRequest.Kind.Kind,
+						"resource_namespace", req.AdmissionRequest.Namespace,
+						"resource_name", req.AdmissionRequest.Name,
+						"dryrun", strconv.FormatBool(r.EnforcementAction == "dryrun"),
```

Comment:
I don't think you need this line since you already encode "deny" or "dryrun" in constraint_action above.

Reply:
ah yes!